src/renderers/WebGLRenderer.js

```diff
@@ -2901,7 +2905,7 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	}
 
-	function setCubeTexture ( texture, slot ) {
+	function setTextureCube ( texture, slot ) {
```

Comment:
I renamed these for consistency reasons. I didn't see why they were named differently.